src/libraries/System.Runtime/README.md

```diff
@@ -0,0 +1,18 @@
+# System.Runtime
+Contains fundamental classes and base classes that define commonly-used value and reference data types, events and event handlers, interfaces, attributes, and processing exceptions.
+
+Documentation can be found here: https://learn.microsoft.com/en-us/dotnet/api/system?view=net-7.0.
+
+TODO should we have a README for System.Private.CoreLib?
```

Comment:
We probably should explain the split here a bit and that many parts live in S.P.Corelib. Same potentially with the other readme's.

Reply:
I agree. Could you take a shot at that blurb? I think you have a better understanding of how things are organized and why.